README.md

```diff
@@ -40,7 +39,7 @@ Node is different because it has this concept called full stack, isomorphic or u
 
```

Comment:
again reference to "I". replace with something like "There's no way Go or some other new platform can compete..."